eq-author/.env.development

```diff
@@ -1,10 +1,2 @@
-REACT_APP_BASE_NAME=""
 REACT_APP_API_URL="http://localhost:4000/graphql"
-REACT_APP_LAUNCH_URL="http://localhost:4000/launch"
-REACT_APP_USE_FULLSTORY="false"
-REACT_APP_USE_SENTRY="false"
-
-REACT_APP_ENABLE_AUTH="true"
```

Comment:
Just wondering why you removed this variable? I normally override it as false to allow me to get past the sign in process locally

Reply:
When the API required auth, we wont be able to use "Guest" anyway so we should all start setting the app up so it works the same for dev as it does when deployed

Reply:
Ok